erica/elster_xml/grundsteuer/elster_data_representation.py

```diff
@@ -3,15 +3,146 @@
 
```

Comment:
It might make sense to split this as well but I wouldn't do it *just* because we split grundsteuer_input. This is a separate "package" with its own unique set of conditions and its structure should not be tightly coupled to that of another package. 

Still, it might be a good idea to split this on its own merit. I will try something and commit it if there's no reason not to.

Reply:
Done.